app/cypress/e2e/0-ndr-core-tests/auth_routes/auth_gp_admin_path_access.cy.js

```diff
@@ -10,48 +10,49 @@ const patient = {
     active: true,
 };
 
-const smokeTest = Cypress.env('CYPRESS_RUN_AS_SMOKETEST') ?? false;
-const baseUrl = Cypress.env('CYPRESS_BASE_URL') ?? 'http://localhost:3000/';
+const baseUrl = Cypress.config('baseUrl');
```

Comment:
cypress will automatically populate the baseUrl if this is set in config, so we can replace all instances of cypress.visit(baseUrl + '/search') or similar with simply cypress.visit('/search').

this would be a much cleaner solution

Reply:
Yes, I think Rio has already replaced `cy.visit(baseUrl + '/route')` by `cy.visit('/route')` in most parts 😄  
I did a search just now and fixed one remaining  at e2e.ts, so we should be free from that now.

For the baseUrl in this file, I think is not for cy.visit but for path comparison  `cy.url().should('eq', baseUrl + '/search/upload');`, which probably still needs the baseUrl provided